omnibus/resources/agent/msi/cal/CustomAction.cpp

```diff
@@ -509,6 +516,8 @@ UINT doUninstallAs(MSIHANDLE hInstall, UNINSTALL_TYPE t)
             DoStartSvc(hInstall, agentService);
         }
     }
+    std::wstring embedded = installdir + L"\\embedded";
+    RemoveDirectory(embedded.c_str());
```

Comment:
What's the behavior of `RemoveDirectory` when the target is a directory symlink? I wasn't able to find information on the Windows docs.

Reply:
it removes the symlink, leaving the contents in the target directory alone.